entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -602,6 +602,20 @@ class EntityService(
         return neo4jRepository.deleteEntity(entityId)
     }
 
+    fun deleteEntityAttribute(entityId: String, attributeName: String, contextLink: String): Boolean {
+        val expandedAttributeName = expandJsonLdKey(attributeName, contextLink)!!
+
+        if (neo4jRepository.hasPropertyOfName(entityId, expandedAttributeName)) {
+            neo4jRepository.deleteEntityProperty(entityId, expandedAttributeName)
+            return true

```

Comment:
you could return something like `deleteEntityProperty(...) >= 1` (and then use the boolean result in the handler to decide on the response : `true` -> 204, `false` -> 400 or 500)

Reply:
Currently, whatever the result of the delete operation, we say it's successful (generally, errors will probably result in an exception but one never knows ...)